src/Simplex/Messaging/Agent/Client.hs

```diff
@@ -1591,9 +1595,11 @@ disableQueuesNtfs :: AgentClient -> [RcvQueue] -> AM' [(RcvQueue, Either AgentEr
 disableQueuesNtfs = sendTSessionBatches "NDEL" 90 id $ sendBatch disableSMPQueuesNtfs
 
 sendAck :: AgentClient -> RcvQueue -> MsgId -> AM ()
-sendAck c rq@RcvQueue {rcvId, rcvPrivateKey} msgId = do
-  withSMPClient c rq ("ACK:" <> logSecret msgId) $ \smp ->
+sendAck c rq@RcvQueue {userId, server, rcvId, rcvPrivateKey} msgId = do
+  withSMPClient c rq ("ACK:" <> logSecret msgId) $ \smp -> do
+    atomically $ incSMPServerStat c userId server ackAttempts
     ackSMPMessage smp rcvPrivateKey rcvId msgId
+    atomically $ incSMPServerStat c userId server ackMsgs
```

Comment:
there can be permanent and temporary errors here too, and also NO_MSG that is better to count separately (e.g. ackDuplicates)